### ISSUE TITLE
Exit before trying second mirror

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/spinalcordtoolbox#1842]
### Description

When manually altering (for the purpose of testing) the URL of mirror 1, the `sct_download_data` crashes before even trying mirror 2. My guess is that this bug has been introduced by Sentry, so maybe we should turn it off during this execution? or altering the `sct.printv(err.message, type='warning')` on line 192 (which should send a "warning" but end up giving an error...?)

~~~
sct_download_data -d deepseg_sc_models

--
Spinal Cord Toolbox (master/51f9f79ffad0cbdb3b865273faa4aa605fced2df)
Running /Users/julien/code/sct/scripts/sct_download_data.py -d deepseg_sc_models

Trying URL: https://osf.io/mg8wr___/?action=download
Traceback (most recent call last):
  File "/Users/julien/code/sct/scripts/sct_download_data.py", line 199, in <module>
    main()
  File "/Users/julien/code/sct/scripts/sct_download_data.py", line 109, in main
    tmp_file = download_data(url, verbose)
  File "/Users/julien/code/sct/scripts/sct_download_data.py", line 172, in download_data
    _, content = cgi.parse_header(response.headers['Content-Disposition'])
  File "/Users/julien/code/sct/python/lib/python2.7/site-packages/requests/structures.py", line 54, in __getitem__
    return self._store[key.lower()][1]
KeyError: 'content-disposition'
Sentry is attempting to send 1 pending error messages
Waiting up to 10 seconds
Press Ctrl-C to quit
Note: you can opt out of Sentry reporting by editing the file ${SCT_DIR}/bin/sct_launcher and delete the line starting with "export SENTRY_DSN"
~~~

**Comment by zougloub** [comment]
What Sentry issue?

**Comment by abtahizadeh** [comment]
The error is caught, but Sentry fails to send, and consequently it does not produce an error, keeps trying ...

**Comment by jcohenadad** [comment]
Fixed via #1843
